pynest/nest/lib/hl_api_info.py

```diff
@@ -102,10 +103,17 @@ def help(obj=None, pager=None):
         Object to display help for
     pager : str, optional
         Pager to use
+    return_text : bool, optional
+        Option for returning the help text
     """
     hlpobj = obj
     if hlpobj is not None:
-        show_help_with_pager(hlpobj, pager)
+        if isinstance(return_text, str):
```

Comment:
@babsey Why would you allow `return_text` to be anything else than `True` or `False`?

Reply:
Your question has a good point. Returning help text was my intention for accessing to nest via web protocol. Here, it requests nest.help with return_text='true' and the type of return_text is string. Thus, it should be changed to boolean type.

By the way, an example for the usage of nest.help via web request can be found here:
http://192.52.2.171/ (this link is temporally available)
This page calls several nest methods e.g. nest.Models, nest.help and nest.GetDefaults and renders to webpage dynamically.

Reply:
Looks nice! I don't know how you web app precisely interfaces with NEST, but I suppose it calls `nest.help()` from Python, and the web app will always call it with `return_value==True`. Why can't you then place the Python constant `True` in the call? I am a bit afraid that details of an external app get too deeply into NEST's code. @jougs @apeyser Any comments?